examples/nextjs-with-auth0-react/cypress/integration/auth.spec.js

```diff
@@ -0,0 +1,124 @@
+/// <reference types="cypress" />
+import { createClient } from '@simulacrum/client';
+import {
+  Button,
+  Heading,
+  TextField,
+  Link,
+  HTML,
+  including,
+} from '@bigtest/cypress';
+
+// need to start up and configure simulation server
+describe('test the login flow', () => {
+  let client = null;
+  let simulation = null;
+  let person = null;
+
+  before(() => {
+    client = createClient('http://localhost:4000');
+  });
+
+  beforeEach(async () => {
+    simulation = await client.createSimulation('auth0', {
+      options: {
+        audience: 'https://your-audience/',
+        scope: 'openid profile read:shows',
+        clientId: 'YOUR_AUTH0_CLIENT_ID',
+        rulesDirectory: './rules',
+      },
+      services: {
+        auth0: {
+          port: 4400, // port for the auth0 service itself
+        },
+      },
+    });
+    let { data } = await client.given(simulation, 'person');
+
+    person = data;
+  });
+
+  afterEach(async () => {
+    if (simulation) {
+      console.log('destroy simulation', simulation);
+      await client.destroySimulation(simulation);
+      simulation = null;
+      person = null;
+    }
+  });
+
+  describe('login from the homepage', () => {
+    beforeEach(() => cy.visit('/'));
+
+    it('login is visible', () => {
+      cy.expect(Button('Login').exists());
+    });
+
+    it.skip('logs in user', () => {
```

Comment:
IMO I don't think we can merge this until all tests pass and nothing is skipped.

Reply:
I agree

Reply:
I was trying to keep the scope of this PR tight and avoiding rabbit holes because I thought it would be helpful to get this in and useful for other PRs.

Alright, we can address this then. I don't know what is causing this, but it may be resolved in the other PRs. It seems like similar behavior? I will see if I can free up a bit of time to dig into this here or in the other PR.